app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -429,8 +449,12 @@ private fun assertDeleteBrowsingDataOnQuitButton() {
 }
 
 private fun assertDeleteBrowsingDataValue(state: String) {
-    mDevice.wait(Until.findObject(By.text("Delete browsing data on quit")), waitingTime)
-    onView(withText(state)).check(matches(withEffectiveVisibility(Visibility.VISIBLE)))
+    onView(
```

Comment:
same, about renaming the methods above: let's apply it here then too

Reply:
Done!